api-reference/20 Data Visualization Widgets/dxPolarChart/1 Configuration/annotations/annotations.md

```diff
@@ -5,7 +5,191 @@ inherits: dxPolarChart.Options.commonAnnotationSettings
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import {WidgetName}, {
+        Annotation
+    } from 'devextreme-react/{widget-name}';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <{WidgetName} ... >
+                    <Annotation
+                        type="text"
+                        text="Annotation text"
+                    />
+                    <Annotation
+                        type="image"
+                        image="http://image/url/myimage.png"
+                    />
+                </{WidgetName}>
+            );
+        }
+    }
+
+    export default App;
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+Annotations can be unattached or anchored to a PolarChart element. The following list shows how to position them. PolarChart coordinates ([argument](/api-reference/_hidden/dxChartCommonAnnotationConfig/argument.md '{basewidgetpath}/Configuration/annotations/#argument'), [value](/api-reference/_hidden/dxChartCommonAnnotationConfig/value.md '{basewidgetpath}/Configuration/annotations/#value'), [axis](/api-reference/_hidden/dxChartCommonAnnotationConfig/axis.md '{basewidgetpath}/Configuration/annotations/#axis'), [series](/api-reference/_hidden/dxPolarChartCommonAnnotationConfig/series.md '{basewidgetpath}/Configuration/annotations/#series')), [angle](/api-reference/_hidden/dxPolarChartCommonAnnotationConfig/angle.md '{basewidgetpath}/Configuration/annotations/#angle') and [radius](/api-reference/_hidden/dxPolarChartCommonAnnotationConfig/radius.md '{basewidgetpath}/Configuration/annotations/#radius') specify the element that the annotation's arrow points to; pixel coordinates ([x](/api-reference/_hidden/dxChartCommonAnnotationConfig/x.md '{basewidgetpath}/Configuration/annotations/#x') and [y](/api-reference/_hidden/dxChartCommonAnnotationConfig/y.md '{basewidgetpath}/Configuration/annotations/#y')) specify the position of the annotation's center.
+
+- **Unanchored annotation**
+
+        annotations: [{
+            x: 100,
+            y: 200
+        }]
+
+- **Annotation anchored to a polar chart coordinate**
+
+        annotations: [{
+            argument: new Date(2019, 1, 16),
+            value: 15
+        }]
+
+- **Annotation anchored to a series point**
+
+        annotations: [{
+            argument: new Date(2019, 1, 16),
+            series: "Series 1"
+        }]
+
+- **Annotation displayed on an axis**
+
+        annotations: [{ 
+            // An annotation on the argument axis 
+            argument: new Date(2019, 1, 16)
+        }, { 
+            // An annotation on the value axis 
+            value: 15
+        }]
+
+- **Mixed anchoring (pixel and chart coordinates used simultaneously)**
+
+        annotations: [{
+            argument: new Date(2019, 1, 16),
+            y: 200
+        }]
+
+- **Annotation's position defined by angle and radius**
```

Comment:
This type of annotations can be called "unanchored" too, only if `x` and `y` specify the position in pixel coordinates, angle and radius specify it in some other coordinates.